src/Core/src/Handlers/ScrollView/ScrollViewHandler.Android.cs

```diff
@@ -246,7 +246,7 @@ Size ICrossPlatformLayout.CrossPlatformMeasure(double widthConstraint, double he
 
 		Size ICrossPlatformLayout.CrossPlatformArrange(Rect bounds)
 		{
-			return VirtualView.CrossPlatformArrange(bounds);
+			return (VirtualView as ICrossPlatformLayout).CrossPlatformArrange(bounds);
```

Comment:
General question, was it a good idea to use `as` throughout this PR? If it was a hard cast you would get a better error message like `InvalidCastException: unable to cast type Foo as type Bar`:
```csharp
return ((ICrossPlatformLayout)VirtualView).CrossPlatformArrange(bounds);
```
Where instead we just get `NullReferenceException` without as much information.